example/lib/assurance.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:async';
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_aepcore/flutter_aepcore.dart';
+import 'package:flutter_aepassurance/flutter_aepassurance.dart';
+
+class assurancePage extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<assurancePage> {
+  String _assuranceVersion = 'Unknown';
+  String _urlText = '';
+
+  @override
+  void initState() {
+    super.initState();
+    initPlatformState();
+  }
+
+  // Platform messages are asynchronous, so we initialize in an async method.
+  Future<void> initPlatformState() async {
+    late String assuranceVersion;
+
+    // Platform messages may fail, so we use a try/catch PlatformException.
+    try {
+      assuranceVersion = await Assurance.extensionVersion;
+    } on PlatformException {
+      log("Failed to get Assurance extension versions");
+    }
+
+    // If the widget was removed from the tree while the asynchronous platform
+    // message was in flight, we want to discard the reply rather than calling
+    // setState to update our non-existent appearance.
+    if (!mounted) return;
+
+    setState(() {
+      _assuranceVersion = assuranceVersion;
+    });
+  }
+
+  // UTIL
+  RichText getRichText(String label, String value) {
+    return new RichText(
+      text: new TextSpan(
+        // Note: Styles for TextSpans must be explicitly defined.
+        // Child text spans will inherit styles from parent
+        style: new TextStyle(
+          fontSize: 14.0,
+          color: Colors.black,
+        ),
+        children: <TextSpan>[
+          new TextSpan(
+              text: label, style: new TextStyle(fontWeight: FontWeight.bold)),
+          new TextSpan(text: value),
+        ],
+      ),
+    );
+  }
```

Comment:
Since the text style is consistent across views, we should extract this in a style class and use it for all views.